code/Style.md

```diff
@@ -118,6 +118,8 @@ Use properly namespaced `*zap.Logger`. By properly namespacing and passing in th
 
 Avoid using global loggers (e.g. `zap.L().Error`, `log.Print`, `fmt.Print`), unless it is `package main`.
 
+__DO NOT LOG__ secrets except that you're doing so with the `DEBUG` level.
```

Comment:
Instead of a negative, how about an affirmative:

> Secrets (database, api, tokens) should only be logged at DEBUG level. Production systems should not be running at DEBUG level, thus keeping secrets out of logs.